src/generic/watcher/types.ts

```diff
@@ -0,0 +1,3 @@
+export const Controller = {
```

Comment:
what does this do?

Reply:
Just a generic name so we dont need to deal with harcoding names.